include/graphblas/base/internalops.hpp

```diff
@@ -4179,6 +4297,9 @@ namespace grb {
 					/** The output domain of this operator. */
 					typedef typename OperatorBase< OP >::D3 D3;
 
+					/** The type of the operator OP. */
+					typedef OP operator_type;
```

Comment:
Nice addition!

Code style though -- should this not be OperatorType? (Classes are CamelCase)